source/user-manual/reference/ossec-conf/active-response.rst

```diff
@@ -46,6 +46,9 @@ Toggles the active-response capability on and off. Setting this option to ``yes`
 .. note::
 
     This option is available on the server, local, and agent installations.
+    
+.. important::
```

Comment:
Currently, the documentation uses either `.. note::` or `.. warning::` and we use as few notes and warnings as possible. Do we need a note or can we explain in the description of this option?